### PR TITLE
Use Composer 1.x on Travis

### PR DESCRIPTION
**Pull request by sanmai**
This PR:

- [x] Changes Travis CI to use Composer 1.x due to pending incompatibility with Box.
- [x] Travis CI's build is green

### CODE REVIEW

**Review by sidz — APPROVED** [review]

**Review by sidz — CHANGES_REQUESTED** [review]
isn't https://github.com/box-project/box/releases/tag/3.9.0 compatible with composer 2?

File: Makefile [hunk 1]
```diff
@@ -15,7 +15,7 @@ help:
 # Variables
 #---------------------------------------------------------------------------
 BOX=./.tools/box
-BOX_URL="https://github.com/humbug/box/releases/download/3.8.5/box.phar"
+BOX_URL="https://github.com/humbug/box/releases/download/3.9.0/box.phar"
```
Comment (sidz) [inline Makefile:18]: it is not available yet: https://github.com/box-project/box/issues/490

Comment (sanmai) [reply Makefile:18]: Yeah, let's wait.

**Comment by sanmai** [comment]
Right, how about we merge this just to fix the build and keep things flowing, but once new Box.phar comes our, we'll revisit this issue?

File: .travis.yml [hunk 2]
```diff
@@ -21,6 +21,7 @@ addons:
       - expect
 
 before_install:
+  - composer self-update --1
```
Comment (sanmai) [inline .travis.yml:24]:
```suggestion
  - composer self-update --1 # v2 needs Box 3.9.0+ 
```

**Review by sidz — APPROVED** [review]

**Comment by sanmai** [comment]
Thank you @sidz

**Pull request merged**